doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -105,10 +62,235 @@ Rather than explaining the same topic multiple times in multiple locations, modu
+* Use simple one-clause sentence structure whenever you can.
+* If you have sentences longer than 25 words, try to break them up or condense them.
+If you can’t, make sure they are in plain English.
+* Avoid using long paragraphs.
+Instead, try to break them into groups of 1-10 lines of text.
+* Place the key information at the beginning of each content element.
+* Supplement your text with pictures only to illustrate complex ideas.
+Avoid redundant imagery.
+Do not use pictures without descriptions.
+* Use numbered lists for step-by-step procedures and bulleted lists for unordered sets of items.
+* Emphasize important points with a bulleted list or use headings.
+* Keep the minimum number of steps for procedure descriptions.
+* Provide links to related content within the documentation to help readers get more information.
+* Provide links to auto-generated documentation (API, etc.).
+
+## Style and tone
+* Avoid using _please_ in instructions unless some step in the procedure causes inconvenience for the user or represents a workaround for some system limitation.
+* Use _sorry_ only to inform about serious software or hardware problems for the user.
+Don’t use _sorry_ if the problem appeared during normal software operation.
+* Do not invoke humor in your content.
+Humor is a culture-specific and subjective matter that doesn't belong in technical content.
+* Do not use subjective words like _simple_, _great_, _superb_, and such.
+Instead, demonstrate that something is simple or superb by providing relevant information.
+* Use short, plain words when possible, avoid formal or elevated style of writing.
+* Don't make up new words or modify their usual meaning.
+* Don't use a more complicated term when there is a simpler one having the exact same meaning.
+* Don’t express opinions about cultures, countries, people, etc.
+* Try to use words that are culturally neutral.
+* Omit needless words.
+Use the minimum number of words required to get the message across.
+* Do not overuse questions.
+If you must use a question, make sure it's relevant to the user and the context.
+Do not make up questions for the sake of creating an imaginary dialog with the user.
+* Do not overuse exclamation points.
+This can make your writing seem emotional to and rather subjective to the user.
+Use exclamation points when they are required to emphasize something really critical and important.
+* Do not use colloquialisms, idioms, sayings, proverbs, etc.
+Keep globalization and cultural considerations in mind.
+* Avoid using non-English words or phrases, such as _ergo_ or _bona fide_, even if you think they are widely used.
+However, there are two exceptions to this rule: _etc._ and _e.g._
+
+## Use of terminology
+* Whenever possible, use common English instead of using terminology.
+Use terminology when you need to be particular and precise about something.
+* Provide term definition on its first mention.
```

Comment:
Add: "Link to the Glossary for terms that are in the Glossary. For others, use **bold** formatting on the first mention in a page."

I know this is also defined in other places. That's fine.